runtime/doc/eval.txt

```diff
@@ -7292,6 +7292,126 @@ search({pattern} [, {flags} [, {stopline} [, {timeout}]]])	*search()*
 		The 'n' flag tells the function not to move the cursor.
 
 
+searchcount([{options}])					*searchcount()*
```

Comment:
`searchcount([{options}])	Dict	get or update search stats`
is missing in the `Builtin Functions` section.

Reply:
Nice catch . Added :)